ddtrace/llmobs/_utils.py

```diff
@@ -27,6 +28,12 @@ def _get_llmobs_parent_id(span: Span) -> Optional[int]:
     return None
 
 
+def _get_span_name(span: Span) -> str:
+    if span.name == LANGCHAIN_APM_SPAN_NAME and span.resource != "":
+        return span.resource.lower()
```

Comment:
Why use lowercase resource name?

Reply:
I thought it looked cleaner, also consistent with all lowercase for our other integration span names (intentional/or per-chance) though I don't have strong feelings here

Reply:
Let's keep it the original resource name as formatting the class/module names might cause confusion. Thanks!